assets/css/climatehub.css

```diff
@@ -1,9 +1,44 @@
-/* Example Section */
-.someCoolClassName {
-  color: black;
-  font-family: Helvetica;
+.navbar-brand.shrink {

```

Comment:
Please add a comment hear like what the example section says. In this case you would write `/* Navbar */` so that we can section our CSS code.